src/sunsynk/definitions3phhv.py

```diff
@@ -266,7 +283,7 @@
 SENSORS += SelectRWSensor(
     142,
     "Load Limit",
-    options={0: "Allow Export", 1: "Essentials", 2: "Zero Export"},
+    options={0: "Allow Export", 1: "Zero Export To Non Essential", 2: "Zero Export To Grid CT"},
```

Comment:
Isn't it "Zero Export To Essential"?

Reply:
from my understanding it's zero export to whatever is between the grid terminals of the inverter and the CT (which should be at the grid connection). Essential is the stuff on the load terminals

Reply:
My 20kW inverter has "Zero Export To Load" and "Zero Export to CT" in the menu. Current "Zero Export" is "Zero Export to CT" in the device vocabulary. So the other "Zero Export to Load" is not "Zero Export To **Non** Essential" but "Zero Export To Essential" in my understanding. In my opinion manufacturer's options are in this case pretty clear.

Reply:
yeah we can keep the manufacturer terminology to not be confusing. There are so many ways to configure the inverter and load management that my assumption can be wrong